### PR TITLE
create religious education subject page

### PR DESCRIPTION
**Pull request by joegibb**

### CODE REVIEW

File: app/views/content/life-as-a-teacher/explore-subjects/religiouseducation.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
```
Comment (gemmadallmandfe) [inline app/views/content/life-as-a-teacher/explore-subjects/religiouseducation.md:1]: I think the URL should be changed to `/explore-subjects/religious-education` - we usually separate distinct words

File: app/views/content/life-as-a-teacher/explore-subjects/religiouseducation/_article.html.erb [hunk 2]
```diff
@@ -0,0 +1,73 @@
+<div class="row inset">
```
Comment (gemmadallmandfe) [inline app/views/content/life-as-a-teacher/explore-subjects/religiouseducation/_article.html.erb:1]: File name will need updating if URL changes

File: app/views/content/life-as-a-teacher/explore-subjects/religiouseducation/_article.html.erb [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+  </section>
+
+  <section class="col col-720">
+    <%= render 'content/shared/quotes/quote_progression_chris' %>
+  </section>
+
+  <section class="col col-720 col-space-l-top">
+    <h2 class="heading--box-blue">Shape your pupils' futures</h2>
+    <div class="inset">
+      <p>Teaching religious education opens the door for your pupils to pursue exciting careers. From humanitarian work and charity work, to the civil service and journalism.</p>
```
Comment (gemmadallmandfe) [inline app/views/content/life-as-a-teacher/explore-subjects/religiouseducation/_article.html.erb:17]:
```suggestion
      <p>Teaching religious education opens the door for your pupils to pursue exciting careers. From humanitarian and charity work, to the civil service and journalism.</p>
```
The second sentence doesn't quite make sense - it's not a full sentence? Suggest looking at other subject pages to see how it is phrased elsewhere?

File: app/views/content/life-as-a-teacher/explore-subjects/religiouseducation/_article.html.erb [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+      <p>Your pupils will learn more from you than you might think. As their religious education teacher, you'll guide your pupils through the skills they need to show their understanding of the rich diversity of beliefs and to engage with big ideas with respect. You will teach them valuable life skills such as empathy, critical thinking and deep reflection.</p>
```
Comment (gemmadallmandfe) [inline app/views/content/life-as-a-teacher/explore-subjects/religiouseducation/_article.html.erb:18]: `to engage **with** big ideas **with** respect` is slightly repetitive so slightly harder to read - is it worth tweaking the wording slightly?

File: app/views/content/life-as-a-teacher/explore-subjects/religiouseducation/_article.html.erb [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+    <p>The religious education syllabus is decided by local councils, academies and faith schools.</p>
+    <p>When you teach 11 to 14 year olds (key stage 3), you might teach pupils to:
+    <ul>
+      <li>investigate morals, ethics and philosophy</li>
+      <li>understand a range of religious and non-religious worldviews</li>
+      <li>identify and investigate a wide range of practices and beliefs</li>
+      <li>articulate beliefs and values</li>
+      <li>respond to questions posed by religious and non-religious views </li>
+    </ul>
+    <p>Skills you'll cover when you teach 14 to 16 year olds (key stage 4) include:</p>
```
Comment (gemmadallmandfe) [inline app/views/content/life-as-a-teacher/explore-subjects/religiouseducation/_article.html.erb:37]: Should this say `**might** include` as well? (I don't know)

**Review by gemmadallmandfe — COMMENTED** [review]
Looks good @joegibb - a few comments for you

File: app/views/content/life-as-a-teacher/explore-subjects/_categories.html.erb [hunk 6]
```diff
@@ -51,6 +51,12 @@
               description: "Find out how to become a physics teacher and inspire the next generation of scientists, engineers and innovators.",
               path: "/life-as-a-teacher/explore-subjects/physics"
             )) %>
+                <%= render Categories::CardComponent.new(heading_tag: "h2", card:
+            OpenStruct.new(
+              title: "Religious education",
+              description: "As a religious education teacher, you will guide pupils through the fascinating and diverse range of beliefs and practices across the world.",
```
Comment (Sarah-DfE) [inline app/views/content/life-as-a-teacher/explore-subjects/_categories.html.erb:57]:
```suggestion
              description: "As a religious education teacher, you'll guide pupils through the fascinating and diverse range of beliefs and practices across the world.",
```

File: app/views/content/life-as-a-teacher/explore-subjects/religiouseducation.md [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+title: Become a religious education teacher
+subject: religious education
+title_paragraph: |-
+  <p>
+  As a religious education (RE) teacher, you have a unique opportunity to guide pupils through the diverse range of beliefs and practices across the world. You’ll challenge them to consider other perspectives and reflect on their own values.</p>
```
Comment (Sarah-DfE) [inline app/views/content/life-as-a-teacher/explore-subjects/religiouseducation.md:6]:
```suggestion
  As a religious education (RE) teacher, you'll have a unique opportunity to guide pupils through the diverse range of beliefs and practices across the world. You’ll challenge them to consider other perspectives and reflect on their own values.</p> 
```

File: app/views/content/life-as-a-teacher/explore-subjects/religiouseducation/_article.html.erb (hunk 4 above)
Comment (Sarah-DfE) [inline app/views/content/life-as-a-teacher/explore-subjects/religiouseducation/_article.html.erb:18]:
```suggestion
      <p>Your pupils will learn more from you than you might think. As their religious education teacher, you'll guide your pupils through the skills they need to show their understanding of the rich diversity of beliefs and to engage with big ideas with respect. You'll teach them valuable life skills such as empathy, critical thinking and deep reflection.</p>
```

File: app/views/content/life-as-a-teacher/explore-subjects/religiouseducation/_article.html.erb [hunk 8]
```diff
@@ -0,0 +1,73 @@
...
+      <li>religious thought and its contribution to the wider world </li>
+      <li>how to engage in respectful debate</li>
+    </ul>
+    <%= render 'content/shared/teaching/ages_abilities' %>
+  </section>
+
+  <section class="col col-space-m col-720">
+    <h2 class="heading--box-blue">Start your journey into teaching religious education</h2>
+    <%= render 'content/shared/qualifications-training/check_qualifcations' %>
+    <p>Religious education teacher training courses have had successful applications from candidates with a range of degrees, including philosophy, psychology, law and history.</p>
```
Comment (Sarah-DfE) [inline app/views/content/life-as-a-teacher/explore-subjects/religiouseducation/_article.html.erb:57]:
```suggestion
    <p>Religious education teacher training courses have had successful applications from candidates with a range of degrees, including philosophy, psychology, law, and history.</p>
```

**Review by Sarah-DfE — COMMENTED** [review]
Just a few mini comments @joegibb - love that we've got  a quote from an RE teacher!

**Review by Sarah-DfE — APPROVED** [review]
Looks good to me!

**Pull request merged**